cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -33,6 +35,27 @@
 
   private static final String MEMORY_KEY = "system.resources.memory";
   private static final String CORES_KEY = "system.resources.cores";
+  private static final String LOG_LEVEL = "system.log.level";
+
+  /**
+   * Set the log level for the {@link LogAppenderInitializer}.
+   *
+   * Same as calling {@link #setLogLevel(Map, LogAppenderInitializer)} with first argument from
+   * {@link Arguments#asMap()}
+   */
+  public static void setLogLevel(Arguments args, LogAppenderInitializer initializer) {
+    setLogLevel(args.asMap(), initializer);
+  }
+
+  /**
+   * Set the log level for the {@link LogAppenderInitializer}.
+   * @param args the arguments to use for looking up resources configurations
+   * @param initializer the LogAppenderInitializer which will be used to set up the log level
+   */
+  public static void setLogLevel(Map<String, String> args, LogAppenderInitializer initializer) {
+    initializer.initialize();
```

Comment:
The reason I asked this is because I don't like util method do more then it should.